include/zephyr/bluetooth/services/ots.h

```diff
@@ -988,6 +1006,25 @@ int bt_ots_client_read_object_metadata(struct bt_ots_client *otc_inst,
 int bt_ots_client_read_object_data(struct bt_ots_client *otc_inst,
 				   struct bt_conn *conn);
 
+/** @brief Write the data of the current selected object.
+ *
+ *  This will trigger an OACP write operation for the current size of the object
+ *  with a specified offset and then expect transfering the content via the L2CAP CoC.
+ *
+ *  The data of the object are returned in the obj_data_read() callback.
+ *
+ *  @param otc_inst     Pointer to the OTC instance.
+ *  @param conn         Pointer to the connection object.
+ *  @param buf          Pointer to the data buffer to be written.
+ *  @param len          Size of data.
+ *  @param offset       Offset to write, usually 0.
+ *  @param mode         Mode: Currently only Truncate or not.
```

Comment:
could you use a named enum, and reference it here?

Reply:
Add enum bt_ots_ocap_write_mode .